### PR TITLE
Add "Submit Review" Popup

### PR DESCRIPTION
**Pull request by Afterwalker99**
**Description**

- Add and integrate submit review popup into main screen.

- **Note:** 
- Currently location & sub-location are hardcoded. We will need to integrate location from the photo in the future. 
- Web call to firebase to submit review & record creation are yet to be implemented.
- Currently, the popup is shown directly after taking the 2nd photo. We will need to move it to the share button when those are implemented.

**To Test:**

1. Run the app.
2. Take a photo and it will should the looks good button.
3. Click on looks good button and take a photo again.
4. Verify the review popup is shown.
5. Verify the design matches the one we had on Figma.
6. Type something into the text box and verify its functionality.
7. Tap on rating stars and verify its functionality.
8. Click on submit review and verify the popup dismisses and the toast "Review Submitted" appears.
9. Repeat 1 - 4.
10. Tap on the cancel button and verify the popup dismisses.

### CODE REVIEW

File: app/src/main/java/com/example/posebook/CameraFragment.kt [hunk 1]
```diff
@@ -83,13 +97,21 @@ class CameraFragment : Fragment(R.layout.fragment_camera) {
...
         object : ImageCapture.OnImageCapturedCallback() {
             @SuppressLint("UnsafeOptInUsageError")
             override fun onCaptureSuccess(imageProxy: ImageProxy) {
-                showPoses();
+                if (!isInitialTaken) {
+                    showPoses()
+                } else {
+                    /* This will be replaced by the function that shows the save and delete button.
+                    For the time being, I will just call the review popup here
+                        */
```
Comment (matekdev) [inline app/src/main/java/com/example/posebook/CameraFragment.kt]: Let's mark with // TODO for future-looking up.

File: app/src/main/java/com/example/posebook/SubmitReviewPopupFragment.kt [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    BottomSheetDialogFragment() {
+    companion object {
+        const val tag = "SubmitReviewPopupFragment"
+    }
+
+    override fun getTheme(): Int {
+        return R.style.RoundedBottomSheetDialog
+    }
+
+    // Temp Placeholder: Web Call to firebase will be added later.
```
Comment (matekdev) [inline app/src/main/java/com/example/posebook/SubmitReviewPopupFragment.kt]: Use // TODO: here

**Review by matekdev — APPROVED** [review]
Nice stuff, works fine. Don't have anything crazy to suggest since android development is outta my knowledge base. Seems reasonable enough.

**Comment by Afterwalker99** [comment]
updated to use todo! thankss!!

**Pull request merged**